extensions/amp-story/1.0/amp-story-affiliate-link.css

```diff
@@ -0,0 +1,175 @@
+  transform-origin: left !important;
+}
+
+@keyframes i-amphtml-story-affiliate-link-expand {
+  0% {
+    width: 40px;
+    border-radius: 50%;
+  }
+  100% {
+    width: 160px;
+    border-radius: 20px;
+  }
+}
+
+@keyframes i-amphtml-story-affiliate-link-collapse {
+  0% {
+    width: 160px;
+    border-radius: 20px;
+  }
+  100% {
+    width: 40px;
+    border-radius: 50%;
+  }
+}
+
+.i-amphtml-story-affiliate-link-circle {
+  position: absolute !important;
+  background: rgba(255, 255, 255, 1) !important;
+  margin: 5px !important;
+  width: 40px !important;
+  height: 40px !important;
+  display: flex !important;
+  justify-content: space-between !important;
+}
+
+[data-affiliate-link-icon][expanded] .i-amphtml-story-affiliate-link-circle {  
+  overflow: hidden !important;
+}
+
+[data-affiliate-link-icon] .i-amphtml-story-reset.i-amphtml-hidden {
+  display: flex !important;
+  justify-content: space-between !important;
+}
+
+.i-amphtml-story-affiliate-link-text {
+  visibility: hidden !important;
+  text-align: center !important;
+  text-overflow: ellipsis !important;
+  overflow: hidden;
+  float: left;
+  white-space: nowrap !important;
+  max-width: 70px !important;
+  width: 70px !important;
+  font-family: 'Roboto', sans-serif !important;
+  font-size: 14px !important;
+  color: #3C4043 !important;
+  letter-spacing: 0 !important;
+  line-height: 40px !important;
+}
+
+.i-amphtml-story-affiliate-link-launch {
+  visibility: hidden !important;
+  background-position: center !important;
+  background-repeat: no-repeat !important;
+  background-size: 18px 18px !important;
+  width: 40px !important;
+  height: 40px !important;
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="18px" height="18px" viewBox="0 0 48 48" fill="%23757575"><path d="M0 0h48v48H0z" fill="none"/><path d="M38 38H10V10h14V6H10c-2.21 0-4 1.79-4 4v28c0 2.21 1.79 4 4 4h28c2.21 0 4-1.79 4-4V24h-4v14zM28 6v4h7.17L15.51 29.66l2.83 2.83L38 12.83V20h4V6H28z"/></svg>') !important;
+}
+
+amp-story-page[active] .i-amphtml-story-affiliate-link-pulse {
+  position: absolute !important;
+  border-radius: 50% !important;
+  background: rgba(255, 255, 255, 0.4) !important;
+  margin: 5px !important;
+  width: 40px !important;
+  height: 40px !important;
+  animation: pulse 1.5s cubic-bezier(0.4, 0.0, 0.2, 1) infinite alternate !important;
+}
+
+@keyframes pulse {
+  0% {
+    transform: scale(1);    
+  }
+  100% {
+    transform: scale(1.3);    
+  }
+}
+
+[data-affiliate-link-icon][expanded] .i-amphtml-story-affiliate-link-pulse {
+  animation: none !important;
+}
+
+[data-affiliate-link-icon][expanded] .i-amphtml-story-affiliate-link-text {
+  visibility: visible !important;
+}
+
+[data-affiliate-link-icon][expanded] .i-amphtml-story-affiliate-link-launch {
+  visibility: visible !important;
+}
+
+[data-affiliate-link-icon]:hover .i-amphtml-story-affiliate-link-circle {
```

Comment:
`:hover` styles are triggered by a tap on touchscreens, making the background always gray on mobile phones / tablets.